ruleset/decoders/0102-fortimail_decoders.xml

```diff
@@ -1,159 +1,189 @@
+<!--
+  Copyright (C) 2015-2021, Wazuh Inc.
+-->
+
+<!-- 
+  Decoders for:
+    FortiMail v6.4.5(GA), build453, 2021.06.09
+
+  Sample logs:
+    2021-07-08T13:45:20.966330-03:00 10.132.128.10 date=2021-07-08 time=13:45:21.846 device_id=XXXXXXXX log_id=0200027993 type=statistics pri=information  session_id="168GjLEk027992-16027992" client_name="" client_ip="11.22.33.44" client_cc="ZZ" dst_ip="12.34.56.78" from="noreply@domain.com" hfrom="noreply@domain.com" to="user@gmail.com" polid="2:2:5:SYSTEM" domain="HHHHHHHH" mailer="mta" resolved="FAIL" src_type="int" direction="out" virus="" disposition="Accept" classifier="Not Spam" message_length="23951" subject="Subject of the message" message_id="896710244.116204.16257.Mail@lnk2489" recv_time="" notif_delay="0" scan_time="0.008128" xfer_time="0.005643" srcfolder="" read_status=""
+-->
+
 <decoder name="fortimail-like">
-   <prematch>time=\S+ device_id=\S+ log_id=\S+ |time=\S+ devid=\S+ log_id=\S+</prematch>
+  <prematch>time=\S+ device_id=\S+ log_id=\S+ |time=\S+ devid=\S+ log_id=\S+</prematch>
+</decoder>
+
+<decoder name="fortimail-like-child">
+  <parent>fortimail-like</parent>
+  <regex>pri=(\w+)</regex>
+  <order>pri</order>
+</decoder>
+
+<decoder name="fortimail-like-child">
+  <parent>fortimail-like</parent>
+  <regex>session_id=(\w+)|session_id="(\.*)"</regex>
+  <order>sessionid</order>
+</decoder>
+
+<decoder name="fortimail-like-child">
+  <parent>fortimail-like</parent>
+  <regex>client_name=(\w+)|client_name="(\.*)"</regex>
+  <order>client.name</order>
 </decoder>
 
 <decoder name="fortimail-like-child">
-   <parent>fortimail-like</parent>
-   <regex>pri=(\w+)</regex>
-   <order>pri</order>
+  <parent>fortimail-like</parent>
+  <regex>client_ip=(\S+)</regex>
+  <order>client.ip</order>
```

Comment:
Should this field be srcip? If we include "client.ip" as a field, we need to modify the Wazuh template. Maybe it should dstip. But don't add a new IP field by now.